test/autoload/device_backend/README.md

```diff
@@ -0,0 +1,13 @@
+## Custom Device Backend for Testing
+
+This is a device backend extension used for testing.
+
+```bash
+test/autoload/device_backend
+├── README.md
+├── backend_pkg               # The backend package
```

Comment:
Would it be possible to reuse the existing modules we already have for testing in `test/cpp_extension/setup.py` to test this without having to create a brand new extension?

Reply:
Thank you so much for your review. I'm testing this idea now.

Reply:
Could you please have a look at this commit https://github.com/pytorch/pytorch/pull/127074/commits/1e7cf76cb4fc7b94aae267c70ba37d75a8b93282